src/components/Apps/AccountListTable/const.js

```diff
@@ -81,6 +81,9 @@ export const accountOtherActions = (vm) => [
   {
     name: 'Clone',
     title: vm.$t('Duplicate'),
+    has: () => {
+      return !vm.asset
+    },
     can: vm.$hasPerm('accounts.add_account') && !vm.$store.getters.currentOrgIsRoot,
     callback: ({ row }) => {
       vm.account = {
```

Comment:
There seem to be some inconsistencies between the proposed changes and current code structure. For instance:

1. In `has` function, it should return true only if vm.asset is false not equal to boolean(true)

2. The `can` function does not match with other functions' signatures.

The above discrepancies necessitate thorough review of all existing conditions and adjustments needed based on your project's needs before making the implementation decisions.